libbeat/_meta/fields.ecs-ext.yml

```diff
@@ -0,0 +1,26 @@
+- key: ecs-ext
+  title: ECS extensions
+  description: >
+    Contains extensions to ECS (https://github.com/elastic/ecs)
```

Comment:
I have added this for common fields that are not (yet) in ECS, but they could be in the future. Changes here should be followed by discussions/PRs in ECS repo.
We could also use the commons file for that, but I think that it's fine to leave the common file for things needed by beats but that don't make sense in ECS.